packages/dev/core/src/Maths/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from "./math.scalar";
-export * from "./math.functions";
+export * as scalar from "./math.functions";

```

Comment:
Why this change ? I do not think it should be here

Reply:
I did this to avoid polluting the `BABYLON` scope. I can revert this if you'd like?

Reply:
yup as it is a breaking change for the functions which were already there, thanks

Reply:
The line is not needed. in UMD the Scalar object should still be present.

Reply:
This is also the reason the build was failing.

Reply:
Fixed in https://github.com/BabylonJS/Babylon.js/pull/15373/commits/df7e2b5b17138058e718aace2bdd5421dae67623.